packages/docs-site/src/content/docs/docs/guides/schema.mdx

```diff
@@ -305,27 +309,46 @@ export default defineDmnoService({
 ### Secrets & security
 Items can be marked as `sensitive: true` and they will be treated accordingly. That means:
 
-- We will mask their values when logging them to the console via the `dmno` CLI
 - They will NOT be exposed via `DMNO_PUBLIC_CONFIG`, only via `DMNO_CONFIG`
+- We will redact their values when logging them to the console via the `dmno` CLI
+- If the `redactSensitiveLogs` service setting is enabled, we will patch global `console` methods redact the value from all logs 
+- If the `interceptSensitiveLeakRequests` service setting is enabled, we will patch global http methods to intercept requests that send it to any domain not on the `allowedDomains` list
 - Depending on the integration, we will help make sure you don't accidentally leak them
-- More amazing security-related features coming soon!
+
+To customize behavior, you can set `sesitive` to an object rather than `true`. Note that an empty object will still mark the item as being sensitive.
```

Comment:
```suggestion
To customize behavior, you can set `sensitive` to an object rather than `true`. Note that an empty object will still mark the item as being sensitive.
```